currency_rate_update_banxico/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
I'm not certain this should be pushed to the repo really

Reply:
This file is created when I run pre-commit run -a 

![image](https://github.com/user-attachments/assets/8dad3943-d51d-453d-ab18-05d73b3c037b)